README.md

```diff
@@ -26,9 +26,9 @@ The workbench is developed by the RNA Bioinformatics Center (RBC). This center i
   - [RNA alignment](#rna-alignment)
   - [RNA annotation](#rna-annotation)
   - [RNA-protein interaction](#rna-protein-interaction)
-  - [Ribosome profiling](#ribosome-profiling)
-  - [RNA-Seq](#rna-seq)
   - [RNA target prediction](#rna-target-prediction)
+  - [RNA-Seq](#ngs---preprocessing)
```

Comment:
I put it this way, just to highlight that there are more tools before RNA-Seq. While using the shortcuts at the beginning for RNA-Seq nothing was happening ... I had to remove the bars | for a proper functionality.